features/verblijfplaats.feature

```diff
@@ -0,0 +1,140 @@
+      | type                              | VerblijfplaatsOnbekend |
+      | verblijfplaatsOnbekend            | true                   |
+      | gemeenteVanInschrijving           | 1999                   |
+      | datumInschrijvingInGemeente.type  | Datum                  |
+      | datumInschrijvingInGemeente.datum | 2022-01-28             |
+
+  Rule: Een verblijfplaats is een adres of locatie in het buitenland wanneer land (13.10) een waarde ongelijk aan '0000' (onbekend) heeft
+    - veld 'type' krijgt de waarde 'VerblijfplaatsBuitenland'
+
+    Abstract Scenario: verblijfplaats buitenland
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                                | waarde        |
+      | gemeenteVanInschrijving (09.10)     | 1999          |
+      | datumInschrijvingInGemeente (09.20) | 20020701      |
+      | land (13.10)                        | <land>        |
+      | datumAanvangAdresBuitenland (13.20) | 20220128      |
+      | adresregel1 (13.30)                 | <adresregel1> |
+      | adresregel2 (13.40)                 | <adresregel2> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                             |
+      | type                | RaadpleegMetBurgerservicenummer    |
+      | burgerservicenummer | 555550001                          |
+      | fields              | burgerservicenummer,verblijfplaats |
+      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'verblijfplaats' gegevens
+      | naam                              | waarde                   |
+      | type                              | VerblijfplaatsBuitenland |
+      | gemeenteVanInschrijving.code      | 1999                     |
+      | datumInschrijvingInGemeente.type  | Datum                    |
+      | datumInschrijvingInGemeente.datum | 2002-07-01               |
+      | land.code                         | <land>                   |
+      | adresregel1                       | <adresregel1>            |
+      | adresregel2                       | <adresregel2>            |
+      | datumVan.type                     | Datum                    |
+      | datumVan.datum                    | 2022-01-28               |
+
+      Voorbeelden:
+      | land | land omschrijving            | adresregel1                 | adresregel2                |
+      | 6014 | Verenigde Staten van Amerika | 1600 Pennsylvania Avenue NW | Washington, DC 20500       |
+      | 9999 | Internationaal gebied        |                             | Amundsen-Scott, Antarctica |
+
+  Rule: Een verblijfplaats is een binnenlands adres wanneer straat (11.10) en/of huisnummer (11.20) een waarde heeft
+    - veld 'type' krijgt de waarde 'Adres' 
+
+    Abstract Scenario: verblijfplaats binnenlands adres
```

Comment:
In de rule is aangegeven straat en/of huisnummer een waarde moeten hebben, maar dat wordt niet geïllustreerd. Ik denk dat het handig is om dat te doen. Ook omdat de scenario al abstract is gemaakt

Reply:
In de woonplaats feature wordt er voor een adres onderscheid gemaakt tussen een BAG en een niet-BAG adres. In deze feature wordt het niet genoemd. Is dit gegeven niet relevant voor bepalen van verblijfplaats type?

Reply:
Ik heb hiervan een abstract scenario gemaakt met voorbeelden van combinaties van straat en huisnummer:
```
      | straat | huisnummer |
      | Spui   | 70         |
      |        | 70         |
      | Spui   |            |
```

Het onderscheid tussen een BAG adres of niet-BAG adres is alleen relevant voor het verblijfplaats type. Dat zou zo zijn geweest als we dat ook aparte types hadden gemaakt.